### DISCUSSION TITLE
Network suddeny stopped working

### DISCUSSION THREAD

**Discussion by preferredindifferent** [utmapp/UTM#5864] [Windows]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
After a reboot, my Win11 VM gets "no Internet connection". When setting the Ethernet connector manually (instead of DHCP) it cannot be pinged or ping devices on the network. Setting to bridge mode seemingly works but cannot communicate with the outside world either.

**Configuration**  
* UTM Version: Version 4.4.4 (92)
* macOS Version: 14.1 (23B74)
* Mac Chip (Intel, M1, ...): M1

**Debug log**  
attached:[debug.log](https://github.com/utmapp/UTM/files/13242446/debug.log)

**Upload VM**  
attached as zip
[config.plist.zip](https://github.com/utmapp/UTM/files/13242504/config.plist.zip)

**Comment by osy** [comment]
Try restarting your Mac.

**Comment by preferredindifferent** [comment]
Unfortunately the issue persists after a reboot (of the Mac, not the VM)...
